homeassistant/components/device_automation/__init__.py

```diff
@@ -209,6 +212,22 @@
     connection.send_result(msg["id"], triggers)
 
 
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "device_automation/action/capabilities",
+        vol.Required("action"): dict,
+    }
+)
+async def websocket_device_automation_get_action_capabilities(hass, connection, msg):
```

Comment:
I wonder if we should have made this a generic websocket command that allows people to pass in a type? ("action", "trigger", "condition")

Reply:
Maybe, and then `device_automation/action/list` etc. should also be replaced by a common `list` command.
Let's keep that in a separate PR though?

Reply:
Yes